engines/bastion/app/assets/javascripts/bastion/components/nutupane.factory.js

```diff
@@ -155,14 +150,14 @@ angular.module('Bastion.components').factory('Nutupane',
                     table.working = false;
                     table.refreshing = false;
                     table.initialLoad = false;
-                });
+                }).catch(function(response) {
+                    table.working = false;
+                    table.refreshing = false;
 
-                if (resourceCall && resourceCall.$promise && resourceCall.$promise.catch) {
```

Comment:
I tried to send the Notification in this block but the 'unhandled promise' error was still being thrown in the browser console. use $promise.then.catch is the correct way to handle this, so I adjusted the code and fixed related test failures.